index.html

```diff
@@ -113,114 +119,189 @@ <h1 class="project-heading">Projects</h1>
+            </button>
+        </div>
+    </div>
+
+
+
+
+
+
+    <section class="about-container">
+        <div class="sub-about">
+            <h1 class="about-heading">About</h1>
+            <p class="content">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+                eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+                minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+                aliquip ea eiusmod
+            </p>
+            <button type="button" class="btn download-button">
+                Get my cv
+                <img src="assert/ic_download.svg" alt="a download button" />
+            </button>
+        </div>
+
         <div class="section-about">
             <!-- Blue about section -->
             <div class="languages blue">
-                <h1>languages</h1>
+                <h1 class="list1">languages</h1>
                 <ul class="container-list">
                     <li><span>&bull; JavaScript(ES5/ES6)</span></li>
                     <li><span>&bull; Ruby</span></li>
                     <li><span>&bull; HTML</span></li>
                     <li><span>&bull; Css</span></li>
                 </ul>
-                <img class="about-img" src="assert/ic_framework.svg" alt="hash tag" />
+                <img class="about-img" src="assert/ic_languages_mob.svg" alt="hash tag" />
             </div>
             <!-- Red about section -->
             <div class="languages red">
-                <h1>Frameworks</h1>
+                <h1 class="list1">Frameworks</h1>
                 <ul class="container-list">
                     <li><span>&bull; Bootstrap</span></li>
                     <li><span>&bull; Ruby on Rails</span></li>
                     <li><span>&bull; RSpec</span></li>
                     <li><span>&bull; Capibara</span></li>
                     <li><span>&bull; Selenuim</span></li>
                 </ul>
-                <img class="about-img" src="assert/ic_skills.svg" alt="framework icon" />
+                <img class="about-img" src="assert/ic_framework.svg" alt="framework icon" />
             </div>
             <!-- Orange about section -->
             <div class="languages orange">
-                <h1>Skills</h1>
+                <h1 class="list1">Skills</h1>
                 <ul class="container-list">
                     <li><span>&bull; Codekit</span></li>
                     <li><span>&bull; Codepen</span></li>
                     <li><span>&bull; Github</span></li>
                     <li><span>&bull; GitLab</span></li>
                     <li><span>&bull; Terminal</span></li>
                 </ul>
-                <img class="about-img" src="/assert/ic_languages_mob.svg" alt="lightbulb icon" />
+                <img class="about-img" src="assert/ic_skills.svg" alt="lightbulb icon" />
             </div>
         </div>
     </section>
 
     <section class="form-section">
```

Comment:
- [OPTIONAL] Nice working on the form section for the desktop version, Kindy fix the below issues before you merge if possible.
- [ ] Kindly check yourname and email `input` fields they are not accepting user input.
- [ ] Kindly align the form so it aligns with the Figma Template you can see screenshots below
- [ ] The mobile version for the form section is not responsive, kindly adjust the length of the `Textarea` message field and the margin for the mobile version

## Figma Template
![contact-1](https://user-images.githubusercontent.com/48371156/166097553-fc529e01-bd4d-46e5-a16d-e3b22f8fa06d.png)

## Your Design
![contact-2](https://user-images.githubusercontent.com/48371156/166097561-f39a1a03-f631-49f7-966e-0a4db6a91d72.png)

## Mobile View

![mobile-c](https://user-images.githubusercontent.com/48371156/166097645-5296299a-b797-4311-970d-78d24e892fd0.png)